src/tseda/datastore.py

```diff
@@ -78,7 +78,8 @@ class SampleSetsTable(Viewer):
     }
 
     create_sample_set_textinput = param.String(
-        doc="Enter name of new sample set. Press Enter (⏎) to create.",
+        doc="""Enter name of new sample set. Press Enter (⏎) to create.
+            If the new sample set does not immediately show up, refresh the page""",
```

Comment:
Added issue #114 about this

Reply:
added a better temporary solution in https://github.com/hannamlmv/tseda/pull/107/commits/77aadf40d84491564a192e37e873fb2b2804576e